phpunit/experimental/interactivity-api/directive-processing-test.php

```diff
@@ -6,16 +6,18 @@
  * @subpackage Interactivity API
  */
 
+// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
```

Comment:
```suggestion
 * @subpackage Interactivity API
 *
 * @phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
 */

```

I.e.: the `disable` comment can be placed in the file docblock, which will leave the file looking cleaner.

I'd still recommend to actually fix this at some point by moving the extraneous class(es) to their own (fxiture ?) file(s).